Planet/CountryDataSource.swift

```diff
@@ -13,7 +13,7 @@ class CountryDataSource {
     
     private let countries: [[Country]]
     
-    init(locale: Locale = .current) {
+    init(locale: Locale = .current, filteredISOCodes: [String] = []) {
```

Comment:
How about we use `Locale.isoRegionCodes` as the default and name the parameter `countryCodes`? This way we can skip the `if` and the `guard` down there.